clickhouse_query/column.py

```diff
@@ -0,0 +1,74 @@
+from utils import ASMixin
+
+
+class TwoArgs:
+    def __init__(self, left_arg, right_arg):
+        self.left_arg = left_arg
+        self.right_arg = right_arg
+
+
+class LT(TwoArgs):
```

Comment:
- There are these classes in `functions.comparison.py` and `functions.in_.py`.
- These classes have [sql injection](https://en.wikipedia.org/wiki/SQL_injection) problem.